networking/src/ModelHelper.ts

```diff
@@ -100,20 +100,18 @@ export abstract class ModelHelper {
         }
 
         const correctAnswer = this.getCorrectAnswer(question)
+        const currentQuestion = gameSession?.questions[gameSession?.currentQuestionIndex ?? 0]
+        let originalScore = gameSession?.teams?.find(teamElement => teamElement.id === team.id)?.score ?? 0 
 
-        return answers!.reduce((score: number, answer: ITeamAnswer | null) => {
-            if (isNullOrUndefined(answer)) {
-                return score
-            }
-
-            if (answer.isTrickAnswer) {
-                return score + this.calculateBasicModeWrongAnswerScore(gameSession, answer.text, question.id)
-            } else {
-                console.log(`${answer.text} === ${correctAnswer?.text} = ${answer.text === correctAnswer?.text}`)
-                return score + (
-                    answer.text === correctAnswer?.text ? this.correctAnswerScore : 0)
-            }
-        }, 0)
+        if (answers.some(answer => answer?.isTrickAnswer===true)){
+          let newScore = originalScore += ModelHelper.calculateBasicModeWrongAnswerScore(gameSession, answers?.find(answer => answer?.isTrickAnswer==true)?.text ?? '', currentQuestion.id)
```

Comment:
Probably cleaner and more readable if you create a new variable that's assigned to the result of `ModelHelper.calculateBasicModeWrongAnswerScore(gameSession, answers?.find(answer => answer?.isTrickAnswer==true)?.text ?? '', currentQuestion.id)` and then just return `originalScore + newVar`